tests/drivers/build_all/testcase.yaml

```diff
@@ -84,3 +84,8 @@ tests:
     extra_args: CONF_FILE=dac.conf
     min_ram: 32
     depends_on: dac spi
+  drivers.modem.hl7800.build:
```

Comment:
rename to `drivers.modem.build`

Reply:
> rename to `drivers.modem.build`

What if I want to add tests for another modem?

Reply:
than we'll do that like we add for different sensors.  Is there something that is mutually exclusive about the modem drivers?